docs/analytics-gtm/pii-remover.md

```diff
@@ -0,0 +1,103 @@
+# Stripping Personally Identifiable Information (PII)
+
+A module exists for stripping out PII from values. It can accept strings, query parameters, arrays or objects. If the value is an array or object, the child values are looped through and then treated as strings.
+
+99% of the code has been taken from the legacy universal analytics PII redaction code.
```

Comment:
I think you can remove this line - it's belongs more in the PR description than the documentation.